src/scss/components/_cards.scss

```diff
@@ -15,7 +15,7 @@ $card-radius                    : 4px !default;
 
 @mixin card() {
 
-    .card {
+    .c-card {
```

Comment:
Not anything to change here, but will need to let the Payment and OrderWeb teams know that this class has changed when it goes in so they can update any relevant classes.

Reply:
Good point, will do.